src/components/GalleryCard/GalleryCardDetails.react.js

```diff
@@ -19,7 +19,7 @@ function GalleryCardDetails({
   avatarURL,
   fullName,
   imagePostDateString,
```

Comment:
@burakakyol just spotted this needs to change to dateString too

Reply:
@jonthomp oh sorry, Actually I updated it , I guess I missed the save file :)